proto/ai/traceable/agent/config/v1/config.proto

```diff
@@ -424,4 +427,13 @@ message ThreadPool {
   // Specifies the maximum duration to wait for a free slot in the buffer in milliseconds.
   // If this interval elapses without finding a free slot, the pending event will be discarded.
   google.protobuf.Int32Value timeout_ms = 4;
+}
+
+message LogsExport {
+  // when true, logs from the agent will be exported to Traceable Platform Agent
+  // via Opentelemetry Logs pipeline. Reporting config is shared with the Traces pipeline.
+  google.protobuf.BoolValue enabled = 1;
+
+  // the level of logs to export, anything below that will be dropped.
```

Comment:
anything below that level will not be exported, it'll only be in the local log file